tests/fixtures/disallow-multiple-var-decl/good/for.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable ember-suave/prefer-destructuring */
```

Comment:
I misread this. The `prefer-destructuring` rule should not complain about `j = arr.length` since we're assigning to a variable of a different name than the property. If it is, it's a bug with the rule.